src/components/asset-list/RecyclerAssetList2/core/RawRecyclerList.tsx

```diff
@@ -52,7 +52,7 @@ const RawMemoRecyclerAssetList = React.memo(function RawRecyclerAssetList({
       onLayout={onLayout}
       ref={ref as LegacyRef<RecyclerListViewRef>}
       refreshControl={<RefreshControl />}
-      renderAheadOffset={ios ? 700 : 2000}
```

Comment:
I didn't find much of a difference between 700 and 1200, as an example, on my phone, if you scroll fast enough you can get blank spaces but it's kinda ok.